resources/views/hardware/view.blade.php

```diff
@@ -600,6 +600,14 @@
                                                     <a href="https://checkcoverage.apple.com/us/{{ \App\Models\Setting::getSettings()->locale  }}/?sn={{ $asset->serial }}" target="_blank">
                                                         <i class="fa-brands fa-apple" aria-hidden="true"><span class="sr-only">Applecare Status Lookup</span></i>
                                                     </a>
+                                                    @elseif ((strtolower($asset->model->manufacturer->name) == "dell") || (str_starts_with(str_replace(' ','',strtolower($asset->model->manufacturer->name)),"dellinc")))
+                                                    <a href="https://www.dell.com/support/home/en-us?app=warranty" target="_blank">
+                                                        <img src="/img/demo/manufacturers/dellicon.png" style="width:25px;height:25px;"><span class="sr-only">Dell Warranty Status Lookup</span></i>
+                                                    </a>
+                                                    @elseif ((strtolower($asset->model->manufacturer->name) == "lenovo") || (str_starts_with(str_replace(' ','',strtolower($asset->model->manufacturer->name)),"lenovoinc")))
+                                                    <a href="https://pcsupport.lenovo.com/us/en/warrantylookup#/" target="_blank">
+                                                        <img src="/img/demo/manufacturers/lenovoicon.png" style="width:25px;height:25px;"><span class="sr-only">Lenovo Warranty Status Lookup</span></i>
```

Comment:
Can I get this `sr-only` text translated (including the one above that I made for Apple).

Something like `'general.warranty_lookup_mfg' => ':manufacturer Warranty Status Lookup',`